### ISSUE TITLE
Allow to specify the file of the script to go down in the data-base

### ISSUE THREAD

**Issue by rubenwefox** [seppevs/migrate-mongo#185]
**Is your feature request related to a problem? Please describe.**
Sometimes I want to go down for an specific script that is not the last one. 

**Describe the solution you'd like**
It could be great to have the option to specify the file you want to go down

**Comment by JensWinkler91** [comment]
I agree with that. It would be great if we could specify which version we could upgrade/downgrade to.

**One possible approach could be:**

_Migration-Files:_
- 20240314162522-0.0.13.js
- 20240314163959-0.0.14.js

_Command_: `migrate-mongo up --description 0.0.14` or `migrate-mongo down --description 0.0.13`

Is there a plan to implement this feature in the future?

**Comment by seppevs** [comment]
Won't fix: users can manually run migrations if needed
